BraintreeCore/src/test/java/com/braintreepayments/api/ConfigurationUnitTest.kt

```diff
@@ -0,0 +1,479 @@
+    }
+
+    @Test
+    fun payPalCurrencyIsoCode_forwardsValueFromConfiguration() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_LIVE_PAYPAL)
+        assertEquals("USD", sut.payPalCurrencyIsoCode)
+    }
+
+    @Test
+    fun isVisaCheckoutEnabled_returnsTrueWhenConfigurationApiKeyExists() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_VISA_CHECKOUT)
+        assertTrue(sut.isVisaCheckoutEnabled)
+    }
+
+    @Test
+    fun visaCheckoutSupportedNetworks_forwardsInvocationToVisaCheckoutConfiguration() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_VISA_CHECKOUT)
+        val expected = listOf("AMEX", "DISCOVER", "MASTERCARD", "VISA")
+        assertEquals(expected, sut.visaCheckoutSupportedNetworks)
+    }
+
+    @Test
+    fun visaCheckoutApiKey_forwardsInvocationToVisaCheckoutConfiguration() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_VISA_CHECKOUT)
+        assertEquals("gwApikey", sut.visaCheckoutApiKey)
+    }
+
+    @Test
+    fun visaCheckoutExternalClientId_forwardsInvocationToVisaCheckoutConfiguration() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_VISA_CHECKOUT)
+        assertEquals("gwExternalClientId", sut.visaCheckoutExternalClientId)
+    }
+
+    @Test
+    fun isGooglePayEnabled_whenGooglePayEnabledInConfig_returnsTrue() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_GOOGLE_PAY)
+        assertTrue(sut.isGooglePayEnabled)
+    }
+
+    @Test
+    fun googlePayAuthorizationFingerprint_forwardsValueFromConfiguration() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_GOOGLE_PAY)
+        assertEquals("google-auth-fingerprint", sut.googlePayAuthorizationFingerprint)
+    }
+
+    @Test
+    fun googlePayEnvironment_forwardsValueFromConfiguration() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_GOOGLE_PAY)
+        assertEquals("sandbox", sut.googlePayEnvironment)
+    }
+
+    @Test
+    fun googlePayDisplayName_forwardsValueFromConfiguration() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_GOOGLE_PAY)
+        assertEquals("Google Pay Merchant", sut.googlePayDisplayName)
+    }
+
+    @Test
+    fun googlePaySupportedNetworks_forwardsValuesFromConfiguration() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_GOOGLE_PAY)
+        val expected = listOf("visa", "mastercard", "amex", "discover")
+        assertEquals(expected, sut.googlePaySupportedNetworks)
+    }
+
+    @Test
+    fun googlePayPayPalClientId_forwardsValuesFromConfiguration() {
+        val sut = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_GOOGLE_PAY)
+        assertEquals("pay-pal-client-id", sut.googlePayPayPalClientId)
+    }
+
+    @Ignore("Update Samsung Pay Configuration to only check configuration instead of checking for `braintree-android-samsung-pay` classes to be present on Java classpath.")
```

Comment:
Are we set to remove Samsung pay in V5?

Reply:
Not sure about this. We do have a merchant asking for DropIn support for it though so I wouldn't be surprised if we keep it.